src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/nonassociations/NonAssociationsMigrationService.kt

```diff
@@ -180,5 +194,8 @@ class NonAssociationsMigrationService(
   }
 }
 
+// Two non-association events occur for each non-association relationship - use the offenderIds to uniquely identify each pair
+fun NonAssociationIdResponse.isNotPrimaryNonAssociation(): Boolean = offenderNo1 > offenderNo2
```

Comment:
not sure why this is a `>` rather than a `!=`?

Reply:
We get 2 events for offender pair - but to ensure we only use one event (to uniquely identify the pair) we decided to only use the offender pair where the first is smaller than the last

Reply:
I assumed that the primary was the one where the first was the same as the second.

Might be better for the comment to say something like
```
// Two non-association events occur for each non-association relationship.
// We call the primary association the one where the first offender number is greater (string comparison) than the second.
```